### PR TITLE
Fixes methods defined on included modules after remountable APIs

### PR DESCRIPTION
**Pull request by myxoh**
Related to the issue raised here:
https://github.com/ruby-grape/grape/issues/1815

Uses the following failing specs:
https://github.com/ruby-grape/grape/pull/1816

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,13 +4,13 @@
...
 * [#1813](https://github.com/ruby-grape/grape/pull/1813): Add ruby 2.5 support, drop 2.2. Update rails version in travis - [@darren987469](https://github.com/darren987469).
-* [#1803](https://github.com/ruby-grape/grape/pull/1803): Adds the ability to re-mount all endpoints in any location - [@myxoh](https://github.com/bschmeck).
+* [#1803](https://github.com/ruby-grape/grape/pull/1803): Adds the ability to re-mount all endpoints in any location - [@myxoh](https://github.com/myxoh).
 * [#1795](https://github.com/ruby-grape/grape/pull/1795): Fix vendor/subtype parsing of an invalid Accept header - [@bschmeck](https://github.com/bschmeck).
 * [#1791](https://github.com/ruby-grape/grape/pull/1791): Support `summary`, `hidden`, `deprecated`, `is_array`, `nickname`, `produces`, `consumes`, `tags` options in `desc` block - [@darren987469](https://github.com/darren987469).
 
 #### Fixes
 
-* Your contribution here.
+* [#1818](https://github.com/ruby-grape/grape/pull/1818): Fixes methods defined on included modules after remountable apis - [@myxoh](https://github.com/myxoh).
```
Comment (dblock) [inline CHANGELOG.md]: Since this is not a change from the previous version, delete this.

File: spec/grape/api_spec.rb [hunk 2]
```diff
@@ -3209,6 +3209,41 @@ def static
           expect { a.mount b }.to_not raise_error
         end
       end
+
+      it 'should correctly include module in nested mount' do
+        module Test
```
Comment (dblock) [inline spec/grape/api_spec.rb]: This may pollute global scope. Either call the module something like `ApiSpecTest` or define it dynamically.

**Comment by andreacfm** [comment]
@myxoh still having issues autoloading helpers like:

```
helpers HelperModule
```

Will try to create a new test case

**Comment by myxoh** [comment]
@andreacfm I've replied to you on the issue. Basically, I can't reproduce that. We can probably create a new test case and PR for that issue

**Comment by andreacfm** [comment]
@myxoh I will try to create a test for the helpers too.

**Comment by myxoh** [comment]
@dblock I think these changes should go in. We should check as soon as we are able to reproducing the loading issues what's going on in a separate PR probably

**Comment by dblock** [comment]
Merged

**Pull request merged**